templates/test/cluster-template-prow-ci-version.yaml

```diff
@@ -166,8 +166,8 @@ spec:
       permissions: "0744"
     - contentFrom:
         secret:
-          key: azure.json
-          name: azure-secret
```

Comment:
This change wasn't done in #802 but the `azure-secret` was removed so assuming this should've been modified as well.